docs/api-reference.md

```diff
@@ -400,20 +400,30 @@ end
 
 ### render
 ```
-render() -> Element | nil
+render(props, state) -> Element | nil
 ```
 
 `render` describes what a component should display at the current instant in time.
 
+```lua
+function MyComponent:render(props, state)
+	-- props == self.props
+	-- state == self.state
```

Comment:
We might want to add a little bit more explanation of what we're trying to convey here. Maybe a little blurb that says that props and state can be accessed either as arguments or as properties on self?

Reply:
I can do that, I was just copying Preact's documentation.